src/Functions.hs

```diff
@@ -193,7 +184,7 @@ _centerMap a s1@[x1, y1] s2@[x2, y2] [o1, o2] =
     (fromx - sx)^2 + (fromy - sy)^2 + (tox - ex)^2 + (toy - ey)^2
 
 centerLine :: ObjFn
-centerLine [LN' l, P' p1, P' p2] _ = 
+centerLine [LN' l, P' p1, P' p2] _ =
```

Comment:
if a function's expecting no params, better to use [] instead of _ so an error happens